benchmarks/avaHockeyChannelentTest/avaHockeyChannel_com1DFACfgPy.ini

```
@@ -110,7 +110,7 @@ tSteps = 0|-1
 #++++++++++++++++ particle Initialisation
 # if true use file (import particles initial distribution from file)
 initialiseParticlesFromFile = True
-particleFile = ../benchmarks/avaHockeySmoothChannelentTest
+particleFile = ../benchmarks/avaHockeyChannelentTest
```

Small change, if it is not too late:
```suggestion
particleFile = ../benchmarks/avaHockeyChannelEntTest
```

---

Done